### DISCUSSION TITLE
App not installed

### DISCUSSION THREAD

**Discussion by Geoff85** [NightscoutFoundation/xDrip#2700] [Q&A] [answered]
Hey guys! 
I am trying to install xdrip+ on a samsung galaxy s22 ultra running Android 13 and keep getting the error "app not installed" when I attempt to install the app. Have tried multiple different builds but no success. Anyone have any ideas? It installed fine on my daughters galaxy s9.

**Comment by Navid200** [comment]
You can install only one version:
https://navid200.github.io/xDrip/docs/DoesNotInstall.html

Are you sure you don't have an existing installation?

**Reply by Geoff85** [reply]
Thanks for the reply Navid. No, there are no other versions of xdrip on the phone.

**Reply by Navid200** [reply]
Please post a screenshot of the error as seen on the phone.

**Reply by Geoff85** [reply]
![Screenshot_20230225_151315_Package installer](https://user-images.githubusercontent.com/126296790/221339703-a8c64b12-3b54-4763-944c-eb58ab90fb0f.jpg)

**Reply by Navid200** [reply]
Would you please go to Android settings.
Then, tap on the search (magnifying glass) symbol.
And type xDrip.

What do you see?

**Comment by Geoff85** [comment]
No results found 😢

________________________________

**Reply by Navid200** [reply]
What do you see right after you tap on the .apk file to install xDrip?

**Reply by Geoff85** [reply]
These are the screens I see as I attempt the install. I have searched my phone and can't find anything that is named xdrip.
Short of a factory resest of the phone I'm not sure what else I can try!


![Screenshot_20230228_064444_Google Play Store](https://user-images.githubusercontent.com/126296790/221697990-2f978090-d712-4b54-9200-7ae55d5c8c25.jpg)
![Screenshot_20230228_064429_Package installer](https://user-images.githubusercontent.com/126296790/221698057-5da21e97-b364-4bd6-bee9-25c470f36c94.jpg) 
![Screenshot_20230228_064434_Package installer](https://user-images.githubusercontent.com/126296790/221698029-0eeb3586-ced8-4361-8613-c27011ca5efb.jpg)

**Reply by Navid200** [reply]
Can you try to install this version?
https://github.com/NightscoutFoundation/xDrip/releases/tag/2022.12.24

**Reply by Geoff85** [reply]
No joy. Same message.  I really do appreciate your efforts! Thanks for sticking with me!

**Reply by jamorham** [reply]
@Geoff85 Is your phone in any way managed by an administrator like your employer or with parental controls? These can block app from sideloading. You could also try disabling play protect but as far as I know here you are able to simply press OK to continue. The way to find out the exact cause of this is via ADB debugging connecting the phone to a PC and looking at the log output to see what is happening with the package manager. It likely will have a reason there for why it is refusing to install. I don't know your level of technical expertise as to how easy this would be for you to achieve.

**Reply by adeel23** [reply]
Hi, I'm having the same issue with my personal Samsung S21+ running android 13. Has anyone managed to get this working?

**Reply by Navid200** [reply]
@adeel23 What is the latest Android patch you have installed on the phone?  Can you find it on your phone and post a screenshot?

It may be under "About phone" under a heading like "Android Version".

**Reply by adeel23** [reply]
@Navid200 I see a heading: Android security patch level: 1 January 2023.  Is this what you're looking for? thanks
![softwareinfo](https://user-images.githubusercontent.com/2759817/222290488-c2bb4f3b-d484-4f5c-a9c7-9f6a5a6e46a1.jpeg)

**Comment by Geoff85** [comment, marked as answer]
Solved! I disabled play protect in the google play store and it worked! Thanks for all your help @Navid200

**Reply by Navid200** [reply]
That is a surprise to me.

I have had to disable Google play protect so that it wouldn't stop xDrip every night at the same time.  But, I've never had to disable it in order to be able to install xDrip.

I'm glad your problem is resolved.
